pylsp_black/plugin.py

```diff
@@ -6,9 +6,11 @@
 import black
 import toml
 from pylsp import hookimpl
+from pylsp._utils import get_eol_chars
```

Comment:
I with the module was `pylsp.utils` and not `_utils`, but not much we can do about that at this point.

Reply:
You're right. We inherited that from the old Palantir project and now it's used everywhere in our fork.